Steam Deck Information

EmeraldWither opened the pull request.

EmeraldWither committed:
Update README.md

CrumblyLiquid commented on README.md:
```diff
@@ -4,6 +4,24 @@ It's possible to run BakkesMod on Linux with an up to date version of Rocket Lea
+### Non-Steam Launchers 
+If you installed Epic Games through Non-Steam Launchers, your wine prefix will be `/home/deck/.steam/steam/steamapps/compatdata/NonSteamLaunchers` (assuming you did not move to an SD Card).
+
+To find the app id, run `protontricks -c Epic` and it will show `Epic Games (APPID)` (assuming you added Epic Games to your Steam Library. The same can be done with Rocket League). 
```
The command should probably be `protontricks -s Epic`

CrumblyLiquid replied:
Also by Non-Steam Launchers I'm assuming you mean https://github.com/moraroy/NonSteamLaunchers-On-Steam-Deck?

CrumblyLiquid commented on README.md:
```diff
@@ -4,6 +4,24 @@ It's possible to run BakkesMod on Linux with an up to date version of Rocket Lea
+You will need to download `protontricks` from flatpak, which can be done through the discover app. Installing `GE-Proton` is also a good idea as well. 
+
+### Non-Steam Launchers 
+If you installed Epic Games through Non-Steam Launchers, your wine prefix will be `/home/deck/.steam/steam/steamapps/compatdata/NonSteamLaunchers` (assuming you did not move to an SD Card).
```
By Non-Steam Launchers I'm assuming you mean https://github.com/moraroy/NonSteamLaunchers-On-Steam-Deck?

CrumblyLiquid commented:
Thanks for the contribution!

The Steam Deck specific things are great!

I'll probably do some more testing and research regarding other launchers, etc. Then I'll add it all in one go.

CrumblyLiquid commented on README.md:
```diff
@@ -4,6 +4,24 @@ It's possible to run BakkesMod on Linux with an up to date version of Rocket Lea
 
+## Steam Deck
+
+You will need to download `protontricks` from flatpak, which can be done through the discover app. Installing `GE-Proton` is also a good idea as well. 
```
`protontricks` probably refers to https://flathub.org/apps/com.github.Matoking.protontricks?

CrumblyLiquid replied:
Is there any specific reason to use Proton-GE over just Proton?

CrumblyLiquid commented on README.md:
```diff
@@ -4,6 +4,24 @@ It's possible to run BakkesMod on Linux with an up to date version of Rocket Lea
+
+To find the app id, run `protontricks -c Epic` and it will show `Epic Games (APPID)` (assuming you added Epic Games to your Steam Library. The same can be done with Rocket League). 
+
+You should also make sure that you import Rocket League into Steam by using the `Find Games` button. 
```
Where is the `Find Games` button located?

CrumblyLiquid replied:
Where is the `Find Games` button located?

EmeraldWither committed:
Create README_steamdeck.md

EmeraldWither committed:
update readme to point to steamdeck readme

EmeraldWither commented:
I just got back from vacation, so that's why I took so long to fix the readme (;-;). 

Anyways, I've moved the Steam Deck instructions into their own readme file, and added full instructions for both RL from Steam, and through Non-Steam Launchers.

Although I have only used the Non-Steam Launchers installation method, I don't see why the RL from Steam installation would be much different. 

Let me know if there is anything you want me to test or change!

CrumblyLiquid commented:
Thanks for the update! I too apologize for my absence.

I've decided that I didn't like the duplication of installation instruction (if they were to change in the future, we would have to change them in both files). So I've created `STEAMDECK.md` that has only the Steam Deck specific bits and references the main `README.md`.

Your instructions were very helpful. Thank you very much for that!

I haven't included the Epic Games (NonSteamLaunchers) approach yet. I want to have as little duplication as possible and I'm not sure how to combine the Steam and Non Steam steps together so I wouldn't have to have multiple sets of instructions for doing essentially the same thing.
